.github/workflows/sycl-linux-precommit.yml

```diff
@@ -45,6 +45,7 @@ jobs:
       build_cache_root: "/__w/"
       build_artifact_suffix: "default"
       build_cache_suffix: "default"
+      build_configure_extra_args: --enable-assertions
```

Comment:
how about we put this in the `ci-defaults` section of the configure.py script instead of yaml?